proposals/2021-05-gravity-bridge-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+
+### Testing
+
+The Althea team is committed to playing a long-term role in upgrading, documenting, and supporting Gravity over the coming years.
+The Gravity bridge is currently live and running in a testnet, which validators can join by following the instructions [here](https://github.com/althea-net/althea-chain/blob/main/docs/althea/testnet-2.md)
+
+### Audit:
+
+The Gravity bridge module is currently undergoing an audit with Informal Systems estimated to be completed by the end of July, 2021.
+
+Phase one of the audit has been completed, which resulted in the addition of evidence based slashing and several other minor design fixes.
+
+The phase two design audit will be completed by the end of June. To be followed by phrase three, an implementation audit to be completed by the end of July.
+
+### Conclusion:
+
+With this proposal, the Althea team, together with Cosmos ecosystem partners, will expedite the development of the Gravity Bridge with an incentivized testnet and launch in Q3 2021. Althea will be closely shepherding the Gravity Bridge throughout all phases related to the testing, audit, and implementation process on the Cosmos Hub.
+
+## Proposers
+
+_The Althea Gravity bridge team._
+
+Deborah Simpiler, Justin Kilpatrick and, Jehan Tremback
+
+We’d like to share praise and thank you for contributions from the following teams!
+
+Interchain Foundation
+
+All in Bits/Tendermint
+
+Sommelier, Informal, Injective, Confio
+
+_Gravity Readiness Committee:_
+
+Justin Kilpatrick and Jehan Tremback, Althea
+
+Zarko Milosevic, Informal Systems
+
+Zaki Manian, Sommelier/Iqlusion
+
+## Governance Votes
+
+The following items summarize the voting options and what it means for this proposal.
+
+- **YES**: You agree that Gravity Bridge should be deployed to the Cosmos Hub.
+- **NO**: You disapprove of deploying Gravity bridge on the Cosmos Hub in its current form (please indicate in the [Cosmos Forum](https://forum.cosmos.network/) why this is the case).
+- **NO WITH VETO**: You are strongly opposed to the deployment of Gravity bridge on the Cosmos Hub and will exit the network if this occurred.
+- **ABSTAIN**: You are impartial to the outcome of the proposal.
+
+## Appendix
+
+### FAQ
+
+### Is running the Gravity Module difficult for Cosmos Validators?
+
+Soliciting feedback from dozens of Cosmos hub validators and over 100 test net participants, we found that running the Gravity module is not a difficult task or undue burden on Cosmos validators.
+
+### Is the Gravity bridge secure?
+
+The Gravity bridge is undergoing an audit by Informal Systems. It will then be up to ATOM holders to interpret the results of the code audit and weigh implementation risks in another governance proposal before deployment. Fundamentally, the design of the Gravity Bridge means that its security is directly represented by the security of the validator set on Cosmos Hub.
```

Comment:
```suggestion
The Gravity bridge is undergoing an audit by Informal Systems. It will then be up to ATOM holders to interpret the results of the code audit and weigh implementation risks in another governance proposal before deployment. Fundamentally, the design of the Gravity Bridge means that its security is directly represented by the security of the validator set on Cosmos Hub. On the Cosmos side, the Gravity Bridge security design adheres to the Cosmos network's ideal of safety over liveness assured by 2/3+ validator voting power, but without sacrificing safety or liveness due to issues that may rarely arise with Ethereum.
```